libraries/AP_Notify/Display.cpp

```diff
@@ -488,11 +488,24 @@ void Display::update_mode(uint8_t r)
 	}
 }
 
+void Display::update_text_empty(uint8_t r)
+{
+    char msg [DISPLAY_MESSAGE_SIZE];
+    memset(msg, ' ', sizeof(msg));
+    draw_text(COLUMN(0), ROW(0), msg);
+}
+
 void Display::update_text(uint8_t r)
 {
     char msg [DISPLAY_MESSAGE_SIZE];
     char txt [NOTIFY_TEXT_BUFFER_SIZE];
 
+    const bool text_is_valid = AP_HAL::millis() - pNotify->_send_text_updated_millis < _send_text_valid_millis;
+    if (!text_is_valid) {
+        update_text_empty(r);
```

Comment:
This *update_text* method is very confusing, but I think that both *_mstartpos* and *_movedelay* should be set to zero in this case.